lightbulb/ext/tasks/triggers.py

```diff
@@ -76,7 +96,23 @@ class CronTrigger(Trigger):
 
         __slots__ = ("_croniter",)
 
-        def __init__(self, crontab: str) -> None:
+        def __init__(
+            self,
+            crontab: str | None = None,
+            /,
+            *,
+            month: _CT = None,
```

Comment:
Why not default the kwargs to asterisks?

Reply:
I'm not 100% what you're asking. The idea is that you pass the `crontab` as an arg, or the others as kwargs.

Reply:
Which you can check through the lack of value passed for `crontab`. Defaulting the kwargs to asterisks just means you can omit the `or` comparisons.

Reply:
I don't really see the benefits of this in this instance but I'll make the change.